configure.ac

```diff
@@ -309,10 +309,12 @@ AC_CHECK_LIB(osxfuse_i64,fuse_new, [FUSE_LIBS="-losxfuse_i64"],
         [ AC_CHECK_LIB(fuse_ino64,fuse_new, [FUSE_LIBS="-lfuse_ino64"],
             [ AC_CHECK_LIB(fuse4x,fuse_new, [FUSE_LIBS="-lfuse4x"],
                 [ AC_CHECK_LIB(fuse,fuse_new, [FUSE_LIBS="-lfuse"],
-                    [ AC_CHECK_LIB(dokanfuse1.dll,fuse_new, [FUSE_LIBS="-ldokanfuse1.dll"],
-                        [AC_MSG_ERROR([
-                            Cannot find fuse library - add alternative library search path using
-                            --with-fuse-lib and rerun configure. ])
+                    [ AC_CHECK_LIB(dokanfuse2.dll,fuse_new, [FUSE_LIBS="-ldokanfuse2.dll"],
+			   [ AC_CHECK_LIB(dokanfuse1.dll,fuse_new, [FUSE_LIBS="-ldokanfuse1.dll"],
+				  [AC_MSG_ERROR([
+				           Cannot find fuse library - add alternative library search path using
+					   --with-fuse-lib and rerun configure. ])
+			   ],)
```

Comment:
Looks like indentation is a bit messed up here?

Reply:
Went weird when I pasted the changes originally and was really out of line a lot

Reply:
Well, simply resolve it and then I think we can merge this PR.

Reply:
Not sure what corrections would need.
Should I do some backspaces?

Reply:
To me it looks like line 313-317 should be indented 10 spaces to the right?

Reply:
This is what I see in notepad ++

![Screenshot_20240912_171528](https://github.com/user-attachments/assets/4b8a37d5-dc62-4cdd-af65-973303d3a5ef)


And this in githubs editor

![Screenshot_20240912_171740](https://github.com/user-attachments/assets/b12836ef-5db7-4210-95ed-e3c9769c1aca)

Reply:
Have you accidentally added TABs instead of spaces perhaps? Because alignment looks wrong also in your example, even though indentation for some reason looks better.